modules/data/src/main/java/webindex/data/fluo/PageObserver.java

```diff
@@ -24,27 +26,28 @@
 import org.apache.fluo.api.data.Bytes;
 import org.apache.fluo.api.data.Column;
 import org.apache.fluo.api.observer.AbstractObserver;
-import org.apache.fluo.recipes.accumulo.export.AccumuloExport;
 import org.apache.fluo.recipes.core.data.RowHasher;
 import org.apache.fluo.recipes.core.export.ExportQueue;
 import org.apache.fluo.recipes.core.map.CollisionFreeMap;
 import org.apache.fluo.recipes.core.types.TypedSnapshotBase.Value;
 import org.apache.fluo.recipes.core.types.TypedTransactionBase;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import webindex.core.Constants;
 import webindex.core.models.Link;
 import webindex.core.models.Page;
+import webindex.core.models.UriInfo;
+import webindex.core.models.update.IndexUpdate;
```

Comment:
is everything in models.update pkg related to exporting?  If so I would prefer calling it models.export... but thats just my personal preference.  Update also make sense, I am just thinking export is more specific.